apps/frontend/src/pages/room/hooks/cursor/useCursorPresence.ts

```
@@ -0,0 +1,67 @@
+import { useCallback, useEffect, useState } from 'react'
+import type { CursorInfoWithId, YjsAwarenessBroadcast } from '@/shared/types'
+
+type Listener = () => void
+
+const listeners = new Set<Listener>()
+let cursors = new Map<string, CursorInfoWithId>()
+
+const emitChange = () => {
+  listeners.forEach(listener => listener())
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)